config/variables.go

```diff
@@ -0,0 +1,171 @@
+	bindAndSet(KeyRedisURI, "REDIS_URI", DefaultRedisURI)
+	bindAndSet(KeyPostgresDSN, "POSTGRES_DSN", DefaultPostgresDSN)
+	bindAndSet(KeyBeaconNodeURIs, "BEACON_URIS", DefaultBeaconURIs)
+	bindAndSet(KeyLogJSON, "LOG_JSON", DefaultLogJSON)
+	bindAndSet(KeyLogLevel, "LOG_LEVEL", DefaultLogLevel)
+
+	// API cmd
+	bindAndSet(KeyAPIListenAddr, "LISTEN_ADDR", APIDefaultListenAddr)
+	bindAndSet(KeyAPIBlockSimURL, "BLOCKSIM_URI", APIDefaultBlockSim)
+	bindAndSet(KeyAPISecretKey, "SECRET_KEY", APIDefaultSecretKey)
+	bindAndSet(KeyAPILogTag, "LOG_TAG", APIDefaultLogTag)
+	bindAndSet(KeyAPIPprofEnabled, "PPROF", APIDefaultPprofEnabled)
+	bindAndSet(KeyAPIInternalAPI, "ENABLE_INTERNAL_API", APIDefaultInternalAPIEnabled)
+	bindAndSet(KeyAPIDebug, "DEBUG_API", APIDefaultDebug)
+
+	// API services
+	bindAndSet(KeyBlockSimMaxConcurrent, "BLOCKSIM_MAX_CONCURRENT", 4)
+	bindAndSet(KeyForceGetHeader204, "FORCE_GET_HEADER_204", false)
+	bindAndSet(KeyDisableBlockPublishing, "DISABLE_BLOCK_PUBLISHING", false)
+	bindAndSet(KeyDisableLowprioBuilders, "DISABLE_LOWPRIO_BUILDERS", false)
+	bindAndSet(KeyNumActiveValidatorProcessors, "NUM_ACTIVE_VALIDATOR_PROCESSORS", 10)
+	bindAndSet(KeyNumValidatorRegProcessors, "NUM_VALIDATOR_REG_PROCESSORS", 10)
+	bindAndSet(KeyGetpayloadRetryTimeoutMs, "GETPAYLOAD_RETRY_TIMEOUT_MS", 100)
+
+	// Website
+	// FIXME: Website & API share the same environment variable
+	bindAndSet(KeyWebsiteListenAddr, "LISTEN_ADDR", WebsiteDefaultListenAddr)
+	bindAndSet(KeyWebsiteShowConfigDetails, "SHOW_CONFIG_DETAILS", WebsiteDefaultShowConfigDetails)
+	bindAndSet(KeyWebsiteLinkBeaconchain, "LINK_BEACONCHAIN", WebsiteDefaultLinkBeaconchain)
+	bindAndSet(KeyWebsiteLinkEtherscan, "LINK_ETHERSCAN", WebsiteDefaultLinkEtherscan)
+	bindAndSet(KeyWebsiteRelayURL, "RELAY_URL", WebsiteDefaultRelayURL)
+	bindAndSet(KeyWebsitePubkeyOverride, "PUBKEY_OVERRIDE", WebsiteDefaultPubkeyOverride)
+
+	// Database
+	bindAndSet(KeyDBDontApplySchema, "DB_DONT_APPLY_SCHEMA", "")
+	bindAndSet(KeyDBTablePrefix, "DB_TABLE_PREFIX", "dev")
+	bindAndSet(KeyDBRunTests, "RUN_DB_TESTS", false)
+	bindAndSet(KeyDBTestDSN, "TEST_DB_DSN", "postgres://postgres:postgres@localhost:5432/postgres?sslmode=disable")
+
+	// Redis
+	bindAndSet(KeyActiveValidatorHours, "ACTIVE_VALIDATOR_HOURS", 3)
+
+	// Beacon
+	bindAndSet(KeyAllowSyncingBeaconNode, "ALLOW_SYNCING_BEACON_NODE", "")
+}
+
+func bindAndSet(key, envVariable string, defaultValue any) {
+	log := common.LogSetup(viper.GetBool("logJSON"), viper.GetString("logLevel"))
+	if err := viper.BindEnv(key, envVariable); err != nil {
+		log.WithError(err).Fatalf("Failed to BindEnv: %s", envVariable)
+	}
+	viper.SetDefault(key, defaultValue)
+	configEnvs[key] = envVariable
+}
+
+// GetConfig returns the key/values for the config
+func GetConfig() map[string]string {
+	config := make(map[string]string)
+	for k, v := range configEnvs {
+		config[v] = viper.GetString(k)
+	}
+	// FIXME: Needs to mask or skip sensitive data
```

Comment:
Is this comment still relevant ?

Reply:
Right now the function is not being used. I'll remove it and leave it for a different PR.